tests/accidental_tests.ts

```diff
@@ -3,122 +3,123 @@
-      Vex.Flow.Test.runTests('Sagittal', Vex.Flow.Test.Accidental.sagittal);
-      test('Automatic Accidentals - Simple Tests', Vex.Flow.Test.Accidental.autoAccidentalWorking);
-      Vex.Flow.Test.runTests('Automatic Accidentals', Vex.Flow.Test.Accidental.automaticAccidentals0);
-      Vex.Flow.Test.runTests(
-        'Automatic Accidentals - C major scale in Ab',
-        Vex.Flow.Test.Accidental.automaticAccidentals1
-      );
-      Vex.Flow.Test.runTests(
-        'Automatic Accidentals - No Accidentals Necsesary',
-        Vex.Flow.Test.Accidental.automaticAccidentals2
-      );
+      Vex.Flow.Test.runTests('Accidental Padding', AccidentalTests.formatAccidentalSpaces);
+      Vex.Flow.Test.runTests('Basic', AccidentalTests.basic);
+      Vex.Flow.Test.runTests('Stem Down', AccidentalTests.basicStemDown);
+      Vex.Flow.Test.runTests('Cautionary Accidental', AccidentalTests.cautionary);
+      Vex.Flow.Test.runTests('Accidental Arrangement Special Cases', AccidentalTests.specialCases);
+      Vex.Flow.Test.runTests('Multi Voice', AccidentalTests.multiVoice);
+      Vex.Flow.Test.runTests('Microtonal', AccidentalTests.microtonal);
+      Vex.Flow.Test.runTests('Microtonal (Iranian)', AccidentalTests.microtonal_iranian);
+      Vex.Flow.Test.runTests('Sagittal', AccidentalTests.sagittal);
+      test('Automatic Accidentals - Simple Tests', AccidentalTests.autoAccidentalWorking);
+      Vex.Flow.Test.runTests('Automatic Accidentals', AccidentalTests.automaticAccidentals0);
+      Vex.Flow.Test.runTests('Automatic Accidentals - C major scale in Ab', AccidentalTests.automaticAccidentals1);
+      Vex.Flow.Test.runTests('Automatic Accidentals - No Accidentals Necsesary', AccidentalTests.automaticAccidentals2);
       Vex.Flow.Test.runTests(
         'Automatic Accidentals - Multi Voice Inline',
-        Vex.Flow.Test.Accidental.automaticAccidentalsMultiVoiceInline
+        AccidentalTests.automaticAccidentalsMultiVoiceInline
       );
       Vex.Flow.Test.runTests(
         'Automatic Accidentals - Multi Voice Offset',
-        Vex.Flow.Test.Accidental.automaticAccidentalsMultiVoiceOffset
+        AccidentalTests.automaticAccidentalsMultiVoiceOffset
       );
-      Vex.Flow.Test.runTests('Factory API', Vex.Flow.Test.Accidental.factoryAPI);
+      Vex.Flow.Test.runTests('Factory API', AccidentalTests.factoryAPI);
     },
-    formatAccidentalSpaces: function (options) {
-      var vf = VF.Test.makeFactory(options, 750, 280);
+    formatAccidentalSpaces: function (options: any) {
+      const vf = VF.Test.makeFactory(options, 750, 280);
       const context = vf.getContext();
-      var softmaxFactor = 100;
+      const softmaxFactor = 100;
       // Create the notes
-      var notes = [
-        new VF.StaveNote({
-          keys: ['e##/5'],
-          duration: '8d',
-        })
-          .addAccidental(0, new VF.Accidental('##'))
+      const notes = [
+        vf
+          .StaveNote({ keys: ['e##/5'], duration: '8d' })
```

Comment:
General comment: I thought we agreed to use the classes directly `new StaveNote`